examples/chart/teleport-kube-agent/.lint/app-and-discovery.yaml

```diff
@@ -0,0 +1,9 @@
+roles: app,kube,discovery
+apps:
+ - name: test
+   uri: https://console.aws.amazon.com/ec2/v2/home
+   labels:
+     env: test
+appResources:
```

Comment:
can you add a test case for this file?

Reply:
It's a bit hard because the config is not rendered directly but in a string in a configmap key, we can't use `assert.sets`. Best I can do is a snapshot.

Reply:
done in https://github.com/gravitational/teleport/pull/35781/commits/7fc079c06301efaf398ab87e49fd275bea78745d